data.json

```
@@ -1546,5 +1546,11 @@
     "githubUsername": "mayankchaudhary26",
     "imageUrl": "https://user-images.githubusercontent.com/56837137/139258468-f7b162b5-94f2-4fb3-94e0-7da43db795dd.png",
     "issueNumber": 805
+  },
+  {
+    "name": "Amitkumar",
```

Shouldn't your name be `Amit Kumar`?
```suggestion
    "name": "Amit Kumar",
```